notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.      ax.plot(muTrue[0], muTrue[1], "x",  markerfacecolor="black", color="k",label='z')
</pre><p>label = '$mean (z)$' make sense?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Hi @karm-patel , actually the figure is too small to add big labels , we would have to reduce the label font size if we increase the text length

Reply:
I see, can you please paste figure, su I can suggest other ways to add legend labels?

Reply:
![image](https://user-images.githubusercontent.com/52831429/168469521-db8cd2ac-eaa3-4e2c-8013-e02bc595c914.png)

Reply:
Okay I'm looking into this, meanwhile can you please paste whole figure so I can give you final go-ahead to create a PR

Reply:
Here is the Gist
[Gist](https://gist.github.com/VibhutiBansal-11/a26ce056f3bb9c433baa9ed61bdafbf4)
![after](https://user-images.githubusercontent.com/52831429/168483866-03843002-f5b5-462c-9a75-445ec6a604a3.png)